orchagent/orch.cpp

```diff
@@ -242,6 +262,20 @@ void Orch::recordTuple(Consumer &consumer, KeyOpFieldsValuesTuple &tuple)
     }
 
     gRecordOfs << getTimestamp() << "|" << s << endl;
+
+    if (gLogRotate)
+    {
+        gLogRotate = false;
+
+        logfileReopen();
+
+        /* double check since reopen could fail */
+
+        if (gRecordOfs.is_open())
+        {
+            gRecordOfs << getTimestamp() << "|" << "#|logrotate on: " << gRecordFile << std::endl;
```

Comment:
ok. I removed the marker for now. Let's see what is the best way to add it later.